packages/cli/snippets/session-auth.js

```diff
@@ -56,19 +56,9 @@ const includeSessionKeyHeader = (request, z, bundle) => {
   return request;
 };
 
-const sessionRefreshIf401 = (response, z, bundle) => {
-  if (bundle.authData.sessionKey) {
-    if (response.status === 401) {
-      throw new z.errors.RefreshAuthError(); // ask for a refresh & retry
-    }
-  }
-  return response;
-};
-
 const App = {
   // ...
   authentication: authentication,
-  beforeRequest: [includeSessionKeyHeader],
-  afterResponse: [sessionRefreshIf401]
```

Comment:
Now that we default to call `throwForStatus` and already on the server-side refresh if the status is 401 and the auth type session or OAuth2 w/refresh, there's no need for this anymore.